locust_cloud/__init__.py

```diff
@@ -170,7 +170,8 @@ def main(locustfiles: list[str] | None = None):
 
         if response.status_code != 200:
             try:
-                logger.error(f"{js['Message']} (HTTP {response.status_code}/{response.reason})")
+                logger.error(js.get("Message", "An unexpected error has occured. Please try again."))
```

Comment:
Can you make it so that we _do_ error-log the status code and .reason if there was no "Message" in the response?